docs/schemas/AgPackage.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$id": "https://w3id.org/traceability/schemas/AgPackage.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$comment": "{\"term\": \"AgPackage\", \"@id\": \"http://www.fao.org/3/x5016e/X5016E04.htm\"}",
+    "title": "AgPackage",
+    "description": "An package (container, crate, etc.) which holds agricultural products. The contents or the package itself should meet the requirements as specified here: https://www.producetraceability.org/documents/FINAL_PTI_Harmonized_Traceabilty_Case_Label_April_2020_(003).pdf",
+    "type": "object",
+    "properties": {
+        "@context": {
+            "type": "array"
+        },
+        "type": {
+            "oneOf": [
+                {
+                    "type": "string"
+                },
+                {
+                    "type": "array"
+                }
+            ]
+        },
+        "packageName": {
+            "$comment": "{\"term\": \"packageName\", \"@id\": \"https://schema.org/name\"}",
+            "title": "Package Name",
+            "description": "Name of the items within the package.",
+            "type": "string"
+        },
+        "grade": {
+            "$comment": "{\"term\": \"grade\", \"@id\": \"https://www.ams.usda.gov/grades-standards\"}",
+            "title": "Food Grade",
+            "description": "Grade of the agricultural products contained within the package.",
+            "type": "string"
+        },
+        "responsibleParty": {
+            "$comment": "{\"term\": \"responsibleParty\", \"@id\": \"https://schema.org/Entity\"}",
+            "title": "Responsible Party",
+            "description": "Entity that produced and packaged the products.",
+            "$ref": "https://w3id.org/traceability/schemas/Entity.json"
+        },
+        "voicePickCode": {
+            "$comment": "{\"term\": \"voicePickCode\", \"@id\": \"https://www.produceinventory.com/files/newsroom/Voice-Pick-Code.pdf\"}",
+            "title": "Voice Pick Code",
+            "description": "A four digit code digest of the GTIN code and Lot Code for use in integrating with voice picking systems.",
+            "type": "string"
+        },
+        "date": {
+            "$comment": "{\"term\": \"date\", \"@id\": \"https://schema.org/DateTime\"}",
+            "title": "Packing, Harvest, or Best By Date",
+            "description": "Human readable date describing the date the package was packed, harvested, or is 'best by.'",
+            "type": "string"
+        },
+        "labelImageUrl": {
```

Comment:
URL in the name is likely correct here, but let's make sure we don't want that to be URI in the future.  I am open on this, but we will be living with whatever this naming convention is for a while